tests/unit/lms/services/oauth2_token_test.py

```diff
@@ -31,16 +31,16 @@ def test_save(self, token_store, db_session, application_instance, lti_user):
             }
         )
 
-    def test_get_returns_token_when_present(self, token_store, oauth_token):
-        result = token_store.get()
+    def test_get_returns_token_when_present(self, svc, oauth_token):
+        result = svc.get()
```

Comment:
Our tests are pretty random about whether svc or the full name is prefered. I think either is fine honestly, but `svc` feels a bit pointlessly short.